.travis.yml

```diff
@@ -65,7 +65,7 @@ jobs:
         - ./bin/run-wp-unit-tests.sh
 
     - name: E2E tests (Admin with plugins) (1/2)
-      env: WP_VERSION=latest POPULAR_PLUGINS=true
+      env: WP_VERSION=latest POPULAR_PLUGINS=true WP_DEBUG=true WP_DEBUG_DISPLAY=true SCRIPT_DEBUG=true
```

Comment:
I will remove this change as soon as Docker run is finished and I will confirm that it also works with Travis.

Reply:
See f949a17.